config/src/config.rs

```diff
@@ -0,0 +1,291 @@
+
+        if self.subgraphs.escrow.config.syncing_interval_secs < Duration::from_secs(10)
+            || self.subgraphs.network.config.syncing_interval_secs < Duration::from_secs(10)
+        {
+            warn!(
+                "Your `syncing_interval_secs` value it too low. \
+                This may overload your graph-node instance, \
+                a recommended value is about 60 seconds."
+            );
+        }
+
+        if self.subgraphs.escrow.config.syncing_interval_secs > Duration::from_secs(600)
+            || self.subgraphs.network.config.syncing_interval_secs > Duration::from_secs(600)
+        {
+            warn!(
+                "Your `syncing_interval_secs` value it too high. \
+                This may cause issues while reacting to updates in the blockchain. \
+                a recommended value is about 60 seconds."
+            );
+        }
+
+        if self.tap.rav_request.timestamp_buffer_secs < Duration::from_secs(10) {
+            warn!(
+                "Your `tap.rav_request.timestamp_buffer_secs` value it too low. \
+                You may discart receipts in case of any synchronization issues, \
+                a recommended value is about 30 seconds."
+            );
+        }
+
+        Ok(())
+    }
+}
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct IndexerConfig {
+    pub indexer_address: Address,
+    pub operator_mnemonic: Mnemonic,
+}
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct DatabaseConfig {
+    pub postgres_url: Url,
+}
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct GraphNodeConfig {
+    pub query_url: Url,
+    pub status_url: Url,
+}
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct MetricsConfig {
+    pub port: u16,
+}
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct SubgraphsConfig {
+    pub network: NetworkSubgraphConfig,
+    pub escrow: EscrowSubgraphConfig,
+}
+
+#[serde_as]
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct NetworkSubgraphConfig {
+    #[serde(flatten)]
+    pub config: SubgraphConfig,
+
+    #[serde_as(as = "DurationSecondsWithFrac<f64>")]
+    pub recently_closed_allocation_buffer_secs: Duration,
+}
+
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct EscrowSubgraphConfig {
+    #[serde(flatten)]
+    pub config: SubgraphConfig,
+}
+
+#[serde_as]
+#[derive(Debug, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct SubgraphConfig {
+    pub query_url: Url,
+    pub deployment_id: Option<DeploymentId>,
+    #[serde_as(as = "DurationSecondsWithFrac<f64>")]
+    pub syncing_interval_secs: Duration,
+}
+
+#[derive(Debug, Deserialize_repr, Clone)]
+#[cfg_attr(test, derive(PartialEq))]
+#[repr(u64)]
+pub enum TheGraphChainId {
+    Ethereum = 1,
+    Goerli = 5,
+    Arbitrum = 42161,
+    Sepolia = 11155111,
```

Comment:
Just to keep the Ethereum chains grouped together
```suggestion
    Goerli = 5,
    Sepolia = 11155111,
    Arbitrum = 42161,
```

Reply:
Done